test/test.xcodeproj/xcshareddata/xcschemes/test.xcscheme

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In order for travis to be able to build, install, test, ect. we need to share our scheme that will be build. This file does that.